ee/clickhouse/sql/sessions/list.py

```diff
@@ -100,3 +97,19 @@
         end_time DESC
     {sessions_limit}
 """
+
+SESSION_EVENTS = """
+SELECT
+    uuid,
+    event,
+    properties,
+    timestamp,
+    elements_chain
+FROM events
+WHERE team_id = %(team_id)s
+  AND event != '$feature_flag_called'
```

Comment:
I wonder why we exclude this type of event but not others, e.g. capture metrics?

Reply:
The list is copied from the above query for consistency.

Not sure which capture metrics you mean but these should not affect any real users.